src/v2/controllers/FormController.js

```diff
@@ -44,7 +44,16 @@ export default BaseLoginController.extend({
     this.listenTo(this.formView, 'save', this.handleFormSave);
   },
 
-  invokeAction (actionPath = '') {
+  invokeAction (options = {}) {
```

Comment:
do you still the following changes?

Reply:
ah, good catch. This can be reverted now.